src/utils/prettyNum.tests.ts

```diff
@@ -0,0 +1,14 @@
+import o from 'ospec';
+import { prettyNum } from './prettyNum';
+
+o.spec('regEscape()', () => {
+	o('Escapes strings', () => {
+		o(prettyNum()).equals('0');
+		o(prettyNum(42)).equals('42');
+		o(prettyNum(123456789)).equals('123.456.789');
+		o(prettyNum(123456.789, 2, '.', ',')).equals('123.456,79');
+		o(prettyNum(1234.56789, 3, ',', '.')).equals('1,234.568');
+		o(prettyNum(123456.789, 2, '', ',')).equals('123456,79');
+		o(prettyNum(123456.789, 2, '', '')).equals('12345679');
```

Comment:
OK, þetta er bara skrýtið. 😄 
Decimal splitter táknið á aldrei að geta verið `''`. Annars er fallið að skila ólæsilegu/röngu úttaki.

Defaultaðu alltaf á punkt - ef ekkert annað er í boði.

```suggestion
		o(prettyNum(123456.789, 2, '', '')).equals('123456.79');
```

Reply:
Þetta er komið